utils/misc.ml

```diff
@@ -416,6 +416,11 @@ module LongString = struct
       set dst (dstoff + i) (get src (srcoff + i))
     done
 
+  let blit_string src srcoff dst dstoff len =
+    for i = 0 to len - 1 do
+      set dst (dstoff + i) (String.get src (srcoff + i))
+    done
```

Comment:
I find this implementation a bit disappointing, as my mental cost model for blitting strings into strings is O(1) rather than O(length), so I would expect `blit_string` to be O(chunk numbers) rather than O(length), but this is not worse than `blit`, and hopefully no one uses LongString blits for performance-critical purposes for now. So the code could stay as it is -- mildly disappointing.

Reply:
I promise to write a faster version when there's a use with `len > 7` :)